drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -158,6 +158,7 @@ apply=Apply
 errorPopoverMessageFailedWithError=The expected value is ''{0}'' but the actual one is ''{1}''.
 errorPopoverMessageFailedWithException=Error message: {0}
 dateFormatPlaceholder=Insert date as yyyy-mm-dd
+dmnDateFormatPlaceholder=Insert date as date("yyyy-mm-dd")
```

Comment:
@jomarko 
Maybe  _Insert date as "yyyy-mm-dd"_  ?

Reply:
The thing is for drools, you need to put in _yyyy-mm-dd_, for DMN you need to put in _date("yyyy-mm-dd")_. The message you propose is already there, however it is true just for drools, not for DMN.

Reply:
@jomarko 
Ok, gotcha